app/code/Magento/InventorySales/etc/di.xml

```diff
@@ -170,4 +170,5 @@
                 type="Magento\InventoryCatalog\Plugin\InventoryConfigurationApi\GetStockItemConfiguration\LoadIsInStockPlugin"/>
     </type>
     <preference for="Magento\InventorySalesApi\Model\GetSkuFromOrderItemInterface" type="Magento\InventorySales\Model\GetSkuFromOrderItem"/>
+    <preference for="\Magento\Sales\Api\OrderRepositoryInterface" type="Magento\InventorySales\Model\OrderRepository" />
```

Comment:
There are a lot of dependencies on the `OrderRepository` implementation and not interface, even in MSI modules. We should take this into account.